Doc/library/sys.rst

```diff
@@ -24,6 +24,17 @@ always available. Unless explicitly noted otherwise, all variables are read-only
 
    .. availability:: Unix.
 
+   .. versionchanged:: next
```

Comment:
When we're in 3.16, I think we'll need to repeat the table we had in 3.14 and make it part of the docs. So you can add a TODO comment:

```rst
.. TODO: move porting recommendations from whatsnew/3.14.rst when we're in Python 3.16
```